vector/src/main/java/im/vector/app/features/notifications/NotificationDrawerManager.kt

```diff
@@ -174,13 +174,13 @@ class NotificationDrawerManager @Inject constructor(
             notificationState.clearAndAddRenderedEvents(eventsToRender)
             val session = currentSession ?: return
             renderEvents(session, eventsToRender)
-            persistEvents(session)
+            persistEvents()
```

Comment:
Is it okay that this code does not run when `currentSession` is `null`?

Reply:
It probably is, since it's used to persist invitations or messages if I'm not mistaken, but it would be great to confirm this with someone with more experience.

Reply:
Yes, I think it's fine.